Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.6.1 as base
```

Comment:
I am okay with switching to the slim version. We would need then to implement a way to install required OS dependencies then on the image for libraries such as nokogiri and Postgres. Are you okay with this?

Reply:
Well, most important to me would be that we take the last version of this ruby branch. Doesn't docker automatically take the last patch level if we only specify minor (2.6. -> 2.6.6)?

Reply:
I think it user tags from dockerhub. Therefore I will check if there is a docker hub tag for 2.6 and then adjust accordingly if there is. Otherwise I will take the approach of selecting the latest supported image. In terms of image size I think we can look at this optimization at a later stage